libs/brig-types/src/Brig/Types/Intra.hs

```diff
@@ -146,28 +147,25 @@ instance ToJSON UserAccount where
 -- NewUserScimInvitation
 
 data NewUserScimInvitation = NewUserScimInvitation
-  { newUserScimInvTeamId :: TeamId,
-    newUserScimInvLocale :: Maybe Locale,
+  { newUserScimDetails :: ScimDetails,
```

Comment:
This won't work: it requires to deploy brig and spar simultaneously, which is impossible.  We need to keep `NewUserScimInvitation` backwards compatible, or introduce a new end-point that has the new behavior, keeping the old end-point alive.

Reply:
Would it be an option to break this PR into two?  One for refactoring, maintaining behavior perfectly (or mostly even) towards the outside, and another one that changes behavior?  I think that would be worth a lot, and we already decided we want to keep the two parts in develop at least as commits.

Reply:
Ah, good catch. I made the NewUserScimInvitation backwards-compatible. With this we can deploy brig first, then spar.

I'm in favor of spearating the PR if not for readability alone. I would however deploy them together.